tools/cli/src/main/java/org/apache/carbondata/tool/DataFile.java

```diff
@@ -443,7 +444,8 @@ void computePercentage(byte[] shardMin, byte[] shardMax) {
      * @return result
      */
     private double computePercentage(byte[] data, byte[] min, byte[] max, ColumnSchema column) {
-      if (column.getDataType() == DataTypes.STRING) {
+      if (column.getDataType() == DataTypes.STRING || column.getDataType() == DataTypes.BOOLEAN
+          || column.hasEncoding(Encoding.DICTIONARY)) {
```

Comment:
dictionary column still have min max right?

Reply:
yes, but min max will be surrogate keys right, showing min and max as dictionary value is not useful right